### ISSUE TITLE
HTML <template> contents are cleared when using "some" plugins

### ISSUE THREAD

**Issue by johanbrook** [lumeland/lume#160]
## The bug

When using a HTML `<template>` element in lume, its contents are cleared when building with a special configuration of plugins.

### Project setup

**src/index.njk**
```html
<!doctype html>
<html lang="en">
    <head>
        <title>lume</title>
        <meta charset="utf-8">
    </head>
    <body>
	<template>
            <h1>Hey</h1>
        </template>
    </body>
</html>
```

**_config.ts**
```ts
import lume from "lume/mod.ts";
import basePath from "lume/plugins/base_path.ts";
import url from "lume/plugins/url.ts";

const site = lume({
  src: "src",
  dest: "build",
});

site
  // Use some random plugins here
  .use(basePath())
  .use(url());

export default site;
```

### Test run

```bash
lume && cat build/index.html
```

### Expected

```
🔥 /index.html /index.njk

🍾 Site built into build
<!doctype html>
<html lang="en">
    <head>
        <title>lume</title>
        <meta charset="utf-8">
    </head>
    <body>
	    <template>
            <h1>Hey</h1>
        </template>
    </body>
</html>
```

### Actual

```
🔥 /index.html /index.njk

🍾 Site built into build
<!DOCTYPE html>
<html lang="en"><head>
        <title>lume</title>
        <meta charset="utf-8">
    </head>
    <body>
	    <template></template>
    

</body></html>
```

### Culprits

In `_config.ts` above, comment out the line `.use(basePath())` and the build will work as expected. Replacing `basePath` with the `inline` plugin will also cause the bug. Not the `date` plugin, however.

## Versions

```bash
$ lume --version
v1.4.2
$ deno --version
deno 1.17.1 (release, x86_64-apple-darwin)
v8 9.7.106.15
typescript 4.5.2
```

***

I thought I was going mad while developing my code, since my `<template>` elements kept going empty on build, whereas they worked all over the rest of the internet 😄😄😄

**Comment by oscarotero** [comment]
Yes, this is an known bug in deno-dom (https://github.com/b-fuze/deno-dom/issues/18) that wasn't fixed yet and I'm not sure when is going to be fixed, it's going slow :(

**Comment by johanbrook** [comment]
Aha! Alpha status things ✨ Then I know, at least. Thanks!

**Comment by oscarotero** [comment]
I think this was fixed in the latest `deno-dom` version, which is included in the latest Lume version (1.7.3).
